### DISCUSSION TITLE
Question about lava buckets

### DISCUSSION THREAD

**Discussion by CommiePenguin75** [TownyAdvanced/Towny#5999] [Q&A] [answered]
So my friend has recently set up a Minecraft server and added some plugins for base Minecraft, he doesn't know a lot about config or server stuff. Anyway, I got on there and decided to speed run a little, found out I can't place lava buckets, nor can my faction leader. My friend said this was a towny error message of "you don't have permission to use lava_bucket" in red letters in chat whenever I placed it, so it wouldn't let me place it, would anyone here be willing to help me understand how to deactivate this or turn off this lava bucket permission? I don't use plugins as I am a mod player so wondering how this plugin works. 
<img width="1016" alt="lavaerrorPNG" src="https://user-images.githubusercontent.com/47642173/177207495-7c3e9454-0e17-4cbc-97a5-eddf544e4e8c.PNG">

**Comment by LlmDl** [comment]
That's not a Towny message. Not sure what plugin makes a message like that.

Edit: Towny messages always start with [Towny].

**Comment by Warriorrrr** [comment, marked as answer]
That message is from Essentials Antibuild, you can probably find an option related to lava buckets in the essentialsx config file.

**Reply by CommiePenguin75** [reply]
Thank you for help, my friend got it, sorry I never use plugins, I use mods so I don't even know what these plugins do lol lol. Thank you for responding in a timely manner. :)
